index.rst

```diff
@@ -6,19 +6,14 @@ Solidity
     :alt: Solidity logo
     :align: center
 
-Solidity is a contract-oriented, high-level language whose syntax is similar to that of JavaScript
-and it is designed to target the Ethereum Virtual Machine (EVM).
+Solidity es un lenguaje de alto nivel orientado a escribir contratos. Su sintaxis es similar a la de JavaScript y está diseñado para enfocarse a la Máquina Virtual de Etehreum (EVM).
```

Comment:
Is it possible to translate `contract-oriented` in the same way as `object-oriented`?